.github/ISSUE_TEMPLATE/api_request.yml

```diff
@@ -0,0 +1,30 @@
+name: API Request
+description: Create an API request for Quilt Standard Libraries.
+labels: [suggestion]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking your time to create your API request! If you want to discuss your feature request, join us on [Discord](https://discord.quiltmc.org).
+
+  - type: textarea
+    id: description
+    attributes:
+      label: Describe how your feature(s) will work if applicable.
+      description: "Describe what you want QSL to implement as clearly as possible. More info means your request is more likely to be accepted."
+
+  - type: textarea
+    id: library
+    attributes:
+      label: Which QSL library should your request be implemented in.
+
+  - type: textarea
+    id: module
+    attributes:
+      label: Which QSL module should your request be implemented in.
```

Comment:
Combine these. I would say `Where does your API fit into QSL's structure? Which library and module? Does this add a new library or module?`